zerver/lib/email_mirror.py

```diff
@@ -200,6 +200,8 @@ class ZulipEmailForwardUserError(ZulipEmailForwardError):
     pass
 
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
+    if len(truncate_topic(topic)) < len(topic):
+        content = " ".join([topic, content])
```

Comment:
I don't think this will make the message visually clear enough - you can find some advice in the discussion on the older PR - https://github.com/zulip/zulip/pull/11057

Reply:
Wait, so there's an open PR on this already? Should I close this one, or just try to make it look like subject line in #11057 ?

Reply:
it's fine to just borrow from the other PR and keep this one open

Reply:
:+1: OK, I just don't wanna be stepping on anyone's toes here